pages/home/content.json

```diff
@@ -12,6 +12,13 @@
   "socialLink": "Follow us",
   "socialLinkTarget": "https://twitter.com/i/notifications",
   "clientsH1": "Our Clients",
+  "clientsCarousel": [
+    "http://tutorial9.s3.amazonaws.com/uploads/2008/12/logo.png",
+    "http://sbifinishing.com/wordpress/wp-content/uploads/2012/12/Logo-no-background.png",
+    "http://wiki.myexperiment.org/images/MyExperiment_logo_5016x960_trans.png",
+    "http://www.masonbruce.com/wp-content/uploads/2015/03/android-logo-transparent-background.png"
+  ],
```

Comment:
The *schema.json* needs to be updated to identify this as a list, otherwise the platform client will get confused.